.github/workflows/gh-actions-pr.yml

```diff
@@ -6,183 +6,367 @@ jobs:
   build:
     name: Build
     runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.allow_failure }}
 
     defaults:
       run:
         shell: bash
 
     strategy:
-      fail-fast: true
+      fail-fast: false
       matrix:
         include:
+          #- FROM:     'ubuntu:lunar'
+          #  COMPILER: 'gcc'
+          #  OpenGL_GL_PREFERENCE: 'LEGACY'
+          #  ENABLE_PIE: 'OFF'
+          #  allow_failure: false
+          #- FROM:     'ubuntu:lunar'
+          #  COMPILER: 'clang'
+          #  OpenGL_GL_PREFERENCE: 'LEGACY'
+          #  ENABLE_PIE: 'OFF'
+          #  allow_failure: false
+          #- FROM:     'ubuntu:lunar'
+          #  COMPILER: 'gcc'
+          #  OpenGL_GL_PREFERENCE: 'GLVND'
+          #  ENABLE_PIE: 'OFF'
+          #  allow_failure: false
+          #- FROM:     'ubuntu:lunar'
+          #  COMPILER: 'clang'
+          #  OpenGL_GL_PREFERENCE: 'GLVND'
+          #  ENABLE_PIE: 'OFF'
+          #  allow_failure: false
+          - FROM:     'ubuntu:kinetic'
```

Comment:
Again, I think we need to restrict our officially supported everything. We are not a large corporation and even there I don't recall supporting so many versions.

Reply:
I agree. The policy we decided on last was to support the LTS versions of distros such as Ubuntu that are still in mainstream support, as well as the latest. We may have to revisit that a bit; it's turning out to be more than I can handle lately.

Reply:
It's generally best effort right now. If we need to drop something because we don't have the bandwidth then that's fine - let's just document it and why.

That said, most of our distros are highly related with some simple packaging changes between them so I don't think it's a huge effort for Linux. Mac/Windows, OTOH, are still in a larger need of support and we could certainly use some help there.

Reply:
Definitely.

Reply:
So Kinetic is no longer supported since Manic is out. Will fix in master.